app/src/main/java/ai/elimu/appstore/BaseApplication.java

```diff
@@ -76,9 +78,22 @@ public DaoSession getDaoSession() {
     public Retrofit getRetrofit() {
         Timber.i("getRetrofit");
 
+        /**
+         * Adding logging interceptor for printing out Retrofit request url
+         * in debug mode
```

Comment:
Is this log setup linked to the Timber configuration in any way?: https://github.com/elimu-ai/appstore/blob/3fcbfab1598e971f8d75cf9c32195cdab6aa08f9/app/src/main/java/ai/elimu/appstore/BaseApplication.java#L35

The Timber logs are also printed in release builds (if warnings errors).

Reply:
Actually we don't need to worry about Timber.
With this interceptor, the request url is automatically printed out for debugging without us having to place any log in our code.